docs/contributions-flow.md

```diff
@@ -14,19 +14,51 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# SwG Contributions Flow
+# SwG Contributions Flow (Experiments ONLY)
 
-This flow allows the publication site to display a flow where users can contribute money to
-the publisher. See [Subscriptions APIs](./core-apis.md).
+This flow allows the publication site to display a flow where users can contribute money to the publisher. See [Contributions APIs](./core-apis.md).
 
+First, please setup the contribution response callback via `setOnContributionResponse`:
 
-The contribution flow will first present a set of amounts user can contribute to.
-A user will get a choice to either select one of the offers, or try request login to claim an existing contribution.
+```
+subscriptions.setOnContributionResponse(function(subscriptionPromise) {
```

Comment:
Can we rename subscriptionPromise to contributionPromise?

Reply:
Done